docs/vidu-docs/参考生视频.md

```
@@ -0,0 +1,186 @@
+  "state": "created",
+  "model": "viduq3-mix",
+  "images": ["https://prod-ss-images.s3.cn-northwest-1.amazonaws.com.cn/vidu-maas/template/reference2video-1.png","https://prod-ss-images.s3.cn-northwest-1.amazonaws.com.cn/vidu-maas/template/reference2video-2.png","https://prod-ss-images.s3.cn-northwest-1.amazonaws.com.cn/vidu-maas/template/reference2video-3.png"],
+  "prompt": "Santa Claus and the bear hug by the lakeside.",
+  "duration": 5,
+  "seed": random_number,
+  "aspect_ratio": "3:4",
+  "resolution": "720p",
+  "credits":credits_number,
+  "created_at": "2025-01-01T15:41:31.968916Z"
+}
+```
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)